server.test.js

```diff
@@ -0,0 +1,66 @@
+const request = require('supertest');
+
+const server = require('./server');
+const db = require('./db');
+
+describe('the route handlers', () => {
+  describe('GET /games', () => {
+    test('responds with a 200 status', async () => {
+      const response = await request(server).get('/games');
+      expect(response.status).toBe(200);
+    });
+    test('responds with json', async () => {
+      const response = await request(server).get('/games');
+      expect(response.type).toMatch(/json/i);
+    });
+    test('always responds with an array, even if there are no games', async () => {
```

Comment:
It's a small thing, but really like how you lifted the description from the spec for this test description 👍